plugins/device/deviceutils/errors.go

```diff
@@ -0,0 +1,10 @@
+package deviceutils
```

Comment:
instead of having a separate file, can we make use of `pkg/error/error.go`?

Reply:
not added in `pkg/error` since this is in `plugins` right now